api/server/src/main/java/javax/websocket/server/ServerEndpointConfig.java

```diff
@@ -268,25 +252,18 @@ public static Builder create(Class<?> endpointClass, String path) {
         private Builder() {
 
         }
+
         /**
-         * Builds the configuration object using the current attributes
-         * that have been set on this builder object.
-         * 
+         * Builds the configuration object using the current attributes that have been set on this builder object.
+         *
          * @return a new ServerEndpointConfig object.
          */
         public ServerEndpointConfig build() {
-            return new DefaultServerEndpointConfig(
-                    this.endpointClass,
-                    this.path,
-                    Collections.unmodifiableList(this.subprotocols),
-                    Collections.unmodifiableList(this.extensions),
-                    Collections.unmodifiableList(this.encoders),
-                    Collections.unmodifiableList(this.decoders),
-                    this.serverEndpointConfigurator
-                 );
+            return new DefaultServerEndpointConfig(this.endpointClass, this.path, this.subprotocols, this.extensions,
+                    this.encoders, this.decoders, this.serverEndpointConfigurator);
```

Comment:
This looks like an API change.
Missing the `Collections.unmodifiableList` changes from master.